ChatApp/Extensions/UIColor+Extension.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  UIColor+Extension.swift
+//  chatApp.test
+//
+//  Created by Ana Dzebniauri on 19.04.23.
+//
+
+import UIKit
+
+extension UIColor {
+    convenience init(red: Int, green: Int, blue: Int, alpha: CGFloat) {
+        self.init(
+            red: CGFloat(red)/255,
+            green: CGFloat(green)/255,
+            blue: CGFloat(blue)/255,
+            alpha: alpha
+        )
+    }
+    
+    //MARK: - Typing Area Colors
+    static let messageTextColor = UIColor(red: 25, green: 25, blue: 25, alpha: 1)
```

Comment:
UIColor is general Type so I think putting some specific color which is only used for messageText does not seems right. In UIColor extension you can but colors that can be used throughout the application and not only for some specific component.